docs/frontend_development.md

```diff
@@ -13,20 +13,23 @@ The Home Assistant frontend is built using web components and powered by the [Po
 
 ### Getting the code
 
-First step is to git clone the [home-assistant-polymer repository][hass-polymer]. You can place the repository anywhere on your system.
+First step is to fork the [home-assistant-polymer repository][hass-polymer] and add the upstream remote. You can place the forked repository anywhere on your system.
 
 ```bash
-$ git clone https://github.com/home-assistant/home-assistant-polymer.git
+$ git clone https://github.com/YOUR_GIT_USERNAME/home-assistant-polymer.git
 $ cd home-assistant-polymer
+$ git remote add upstream https://github.com/home-assistant-polymer/home-assistant-polymer.git
 ```
 
 ### Configuring Home Assistant
 
+You will need to have an instance of Home Assistant set up. For example see [macOS installation instructions](https://www.home-assistant.io/docs/installation/macos/).
```

Comment:
We should link to our HA dev instructions.

Reply:
Do you mean [Starting with Development](https://developers.home-assistant.io/docs/en/development_index.html)? That page doesn't really explain the "you need a HA server running" part. I've changed it to Getting Started guide how's that?